### PR TITLE
docs/politis: add settings section

### PR DESCRIPTION
**Pull request by p-nedelchev**

### CODE REVIEW

**Comment by mgenov** [comment]
Here are some latest changes:
image

File: src/pages/politis/guide/settings/settings.bg.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+date: "2020-03-04"
+title: Настройки
```
Comment (mgenov) [inline src/pages/politis/guide/settings/settings.bg.md]: It is better to be named: "Общи Настройки" regarding the latest changes:

image

Comment (p-nedelchev) [reply src/pages/politis/guide/settings/settings.bg.md]: Changes were made according to recent changes. 
image

**Review by mgenov — APPROVED** [review]
LGTM!

**Pull request merged**